content/install_maintain/agents/configuration.md

```diff
@@ -78,6 +78,7 @@ Name                 | Type        | Description
 `key`                | string      | For host agents that are installed via SSH, this may be either the path to the private key that will be used to connect to the host, or the actual private key (beginning with "`-----BEGIN RSA PRIVATE KEY-----`").
 `password`           | string      | For host agents that are installed via SSH (on Linux) and WinRM (on Windows), this property can be used to connect to the host. <br> For Linux hosts, this property is optional if the `key` property is correctly configured. <br> For Windows hosts that are installed via WinRM, this property is also optional and depends on whether the `password` runtime property has been set by the relevant IaaS plugin, prior to agent installation.
 `port`               | integer     | For host agents that are installed via SSH (on Linux) and WinRM (on Windows), this is the port used to connect to the host. <br> The default values are `22` for Linux hosts and `5985` for Windows hosts.
+`transport`          | string      | For Windows agents installed with the `remote` installatino method only: defines the WinRM transport to use (valid values are outlined here: {{< field "pywinrm_transport_link" >}})
```

Comment:
you can use the syntax `[label]({{< field "pywinrm_transport_link" >}})` to add a label to the link (eg. "valid values are outlined *here*" with the word here being the link)